extensions/contract-termination/src/main/java/de/sovity/edc/extension/contacttermination/MapperUtils.java

```diff
@@ -0,0 +1,28 @@
+/*
+ *  Copyright (c) 2024 sovity GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       sovity GmbH - initial API and implementation
+ *
+ */
+
+package de.sovity.edc.extension.contacttermination;
+
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class MapperUtils {
```

Comment:
A very generic name for a public class in a module imported by others

How about "ContractTerminationMapper" and method name `buildTerminationRequest` (in sync with that we should not have some class that is called *the* process without it being clear, what it actually does)

Reply:
we do not have any sort of model space, we have very concern/purpose driven DTOs. For there to be a "Model"-Perspective we'd need a large enough model space, e.g. for a simulation. But we're in backend developments. Unless locally, we have Request, Responses, Messages and maybe Parameter Objects for specific calls. We do not profit from any "Model" perspective.

This is a class in this module, DRY/WET, write everything twice. We're not creating an abstraction here, but need a place for a very specific mapping method.

Reply:
also, out of experience, trying to create this is problematic, it's only worth it when you have a huge surface to map between, that's completely similar. We just need a few methods here, so we should not start a pattern, but have a named unit of code, named to what it actually does.

Reply:
Class removed, function renamed, method moved to the 1 location where it was used.